src/class_sqlThread.py

```diff
@@ -29,22 +18,144 @@
     import tr
     from bmconfigparser import BMConfigParser
     from debug import logger
-    # pylint: disable=attribute-defined-outside-init,protected-access
     from addresses import encodeAddress
+except ImportError:
+    from . import helper_sql
+    from . import helper_startup
+    from . import paths
+    from . import queues
+    from . import state
+    from . import tr
+    from .bmconfigparser import BMConfigParser
+    from .debug import logger
+    from .addresses import encodeAddress
 
+# pylint: disable=attribute-defined-outside-init,protected-access
+root_path = os.path.dirname(os.path.dirname(__file__))
 
-class sqlThread(threading.Thread):
+
+def connection_build():
+    """
+        Stablish SQL connection
+    """
+    conn = sqlite3.connect(state.appdata + 'messages.dat')
+    conn.text_factory = str
+    cur = conn.cursor()
+    return conn, cur
+
+
+class UpgradeDB(object):
+    """
+        Upgrade Db with respect to versions
+    """
+
+    conn, cur = connection_build()
```

Comment:
here allow to call it optionally with a parameter for in memory database

Reply:
pass arguments with DB file name

Reply:
Done